modules/Expectation.js

```diff
@@ -293,77 +293,47 @@ class Expectation {
     return this
   }
 
-  toInclude(value, compareValues, message) {
+  toInclude(value, compareValues, message, _exclude = false) {
```

Comment:
Definitely not a fan of the `_exclude` param - this adds an obscure boolean param to the public API. Could we refactor both `toInclude` and `toExclude` to call into a private (closed-over) function instead?